### PR TITLE
fix some typo in docs

### PR DESCRIPTION
**Pull request by garanews**
### Description

fix some typo in docs

### Motivation and Context
singed vs signed
succeding vs succeeding 
fileter vs filter
kernal vs kernel
libary vs library

### CODE REVIEW

File: docs/ContribOperators.md [hunk 1]
```diff
@@ -3716,9 +3716,9 @@ This version of the operator has been available since version 1 of the 'com.micr
 <dt><tt>char_embedding_size</tt> : int</dt>
 <dd>Integer representing the embedding vector size for each char.If not provide, use the char embedding size of embedding vector.</dd>
 <dt><tt>conv_window_size</tt> : int</dt>
-<dd>This operator applies convolution to word from left to right with window equal to conv_window_size and stride to 1.Take word 'example' for example, with conv_window_size equal to 2, conv is applied to [ex],[xa], [am], [mp]...If not provide, use the first dimension of conv kernal shape.</dd>
+<dd>This operator applies convolution to word from left to right with window equal to conv_window_size and stride to 1.Take word 'example' for example, with conv_window_size equal to 2, conv is applied to [ex],[xa], [am], [mp]...If not provide, use the first dimension of conv kernel shape.</dd>
```
Comment (tianleiwu) [inline docs/ContribOperators.md:3719]: Also update here:
https://github.com/microsoft/onnxruntime/blob/dc486d146b125db780fa2b71eda16875d01cd5e0/onnxruntime/core/graph/contrib_ops/contrib_defs.cc#L1766

File: docs/ContribOperators.md [hunk 2]
```diff
@@ -1786,7 +1786,7 @@ This version of the operator has been available since version 1 of the 'com.micr
 ### <a name="com.microsoft.LongformerAttention"></a><a name="com.microsoft.longformerattention">**com.microsoft.LongformerAttention**</a>
 
   Longformer Self Attention with a local context and a global context. Tokens attend locally: Each token
-  attends to its W previous tokens and W succeding tokens with W being the window length. A selected few tokens
+  attends to its W previous tokens and W succeeding tokens with W being the window length. A selected few tokens
```
Comment (tianleiwu) [inline docs/ContribOperators.md:1789]: This file is auto generated by tool.

Shall also change the source: https://github.com/microsoft/onnxruntime/blob/0c6037b5abe571fc43a55ef7a9d2f846820fbe5d/onnxruntime/core/graph/contrib_ops/bert_defs.cc#L103

File: docs/ContribOperators.md [hunk 3]
```diff
@@ -2670,7 +2670,7 @@ This version of the operator has been available since version 1 of the 'com.micr
 
 <dl>
 <dt><tt>T</tt> : tensor(uint8), tensor(int8)</dt>
-<dd>Constrain input and output types to singed/unsigned int8 tensors.</dd>
+<dd>Constrain input and output types to signed/unsigned int8 tensors.</dd>
```
Comment (tianleiwu) [inline docs/ContribOperators.md:2673]: Update the source as well:
https://github.com/microsoft/onnxruntime/blob/0c6037b5abe571fc43a55ef7a9d2f846820fbe5d/onnxruntime/core/graph/contrib_ops/nhwc_schema_defs.cc#L74

File: docs/ContribOperators.md [hunk 4]
```diff
@@ -2912,7 +2912,7 @@ This version of the operator has been available since version 1 of the 'com.micr
...
```
Comment (tianleiwu) [inline docs/ContribOperators.md:2915]: Update the source as well:
https://github.com/microsoft/onnxruntime/blob/6efa9d9e106c818020215988046aec86a829baf5/onnxruntime/core/graph/contrib_ops/quantization_defs.cc#L615

File: docs/ContribOperators.md [hunk 5]
```diff
@@ -3716,9 +3716,9 @@ This version of the operator has been available since version 1 of the 'com.micr
 <dt><tt>char_embedding_size</tt> : int</dt>
 <dd>Integer representing the embedding vector size for each char.If not provide, use the char embedding size of embedding vector.</dd>
 <dt><tt>conv_window_size</tt> : int</dt>
-<dd>This operator applies convolution to word from left to right with window equal to conv_window_size and stride to 1.Take word 'example' for example, with conv_window_size equal to 2, conv is applied to [ex],[xa], [am], [mp]...If not provide, use the first dimension of conv kernal shape.</dd>
+<dd>This operator applies convolution to word from left to right with window equal to conv_window_size and stride to 1.Take word 'example' for example, with conv_window_size equal to 2, conv is applied to [ex],[xa], [am], [mp]...If not provide, use the first dimension of conv kernel shape.</dd>
 <dt><tt>embedding_size</tt> : int</dt>
-<dd>Integer representing the embedding vector size for each word.If not provide, use the fileter size of conv weight</dd>
+<dd>Integer representing the embedding vector size for each word.If not provide, use the filter size of conv weight</dd>
```
Comment (tianleiwu) [inline docs/ContribOperators.md:3721]: The source is here:
https://github.com/microsoft/onnxruntime/blob/dc486d146b125db780fa2b71eda16875d01cd5e0/onnxruntime/core/graph/contrib_ops/contrib_defs.cc#L1759

**Review by faxu — APPROVED** [review]

File: docs/ContribOperators.md [hunk 6]
```diff
@@ -1786,7 +1786,7 @@ This version of the operator has been available since version 1 of the 'com.micr
 ### <a name="com.microsoft.LongformerAttention"></a><a name="com.microsoft.longformerattention">**com.microsoft.LongformerAttention**</a>
 
   Longformer Self Attention with a local context and a global context. Tokens attend locally: Each token
-  attends to its W previous tokens and W succeding tokens with W being the window length. A selected few tokens
```
Comment (faxu) [inline docs/ContribOperators.md:1789]: This page is generated and should not be manually modified.

File: docs/python/inference/api_summary.rst [hunk 7]
```diff
@@ -265,7 +265,7 @@ Internal classes
 ----------------
 
 These classes cannot be instantiated by users but they are returned
-by methods or functions of this libary.
+by methods or functions of this library.
```
Comment (faxu) [inline docs/python/inference/api_summary.rst:268]: @xadupre Can you confirm this file can be directly modified?

**Review by faxu — CHANGES_REQUESTED** [review]

File: docs/python/inference/api_summary.rst (hunk 7 above)
Comment (xadupre) [reply docs/python/inference/api_summary.rst:268]: Yes, every file .rst in the repo can be manually modified. If their are not in the repo, they are automatically generated.

File: docs/ContribOperators.md (hunk 6 above)
Comment (tianleiwu) [reply docs/ContribOperators.md:1789]: Use tool to generate this doc will likely get the same result. Do we have test in pipeline to verify this?

Comment (faxu) [reply docs/ContribOperators.md:1789]: The Windows GPU CI pipeline covers this, and I confirmed that this change is ok.

**Review by faxu — APPROVED** [review]

**Pull request merged**